demos/src/pagination.mustache

```diff
@@ -7,14 +7,14 @@
 	<button class="o-buttons">25</button>
 	<button class="o-buttons">26</button>
 	<button class="o-buttons">27</button>
-	<button class="o-buttons"><i class="o-ft-icons-icon o-ft-icons-icon--arrow-right"></i></button>
+	<button class="o-buttons o-buttons-icon o-buttons-icon--arrow-right o-buttons-icon--icon-only"><span class='o-buttons-icon__label'>More results</span></button>
 </div>
 
 <br />
 <br />
 
 <div class="o-buttons__pagination">
-	<button class="o-buttons o-buttons--big" disabled="disabled"><i class="o-ft-icons-icon o-ft-icons-icon--arrow-left"></i></button>
+	<button class="o-buttons o-buttons--big o-buttons-icon o-buttons-icon--big o-buttons-icon--arrow-left" disabled="disabled"><span class='o-buttons-icon__label'>Fewer results</span></button>
```

Comment:
Just tested this out locally and the two left arrow icon buttons in "big" and "grouped small" only for some reason, seem to require the `o-buttons-icon--icon-only` class to centre the icon.